langlib/lang.__internal/src/main/ballerina/src/lang.__internal/stream-utils.bal

```diff
@@ -14,6 +14,8 @@
 // specific language governing permissions and limitations
 // under the License.
 
+import ballerina/lang.java;
```

Comment:
Not sure this is the correct thing to do. I think the java module should remain as `ballerina/java` @sameerajayasoma @Kishanthan  WDYT?

Reply:
It can still remain as `ballerina/java`. Since this module is moved inside the langlib dir, I made the module name as such. But it is not the final thing and we need to decide on that.

Reply:
Yes, `ballerina/lang.*` is reserved for langlib modules. Let's continue to use `ballerina/java`